std/haxe/web/Scheme.hx

```diff
@@ -0,0 +1,57 @@
+package haxe.web;
+/**
+* The scheme consists of a sequence of characters beginning with a letter and followed by any combination of letters, digits, plus (+), period (.), or hyphen (-). Although schemes are case-insensitive, the canonical form is lowercase and documents that specify schemes must do so with lowercase letters. It is followed by a colon (:).
+* @class Scheme
+* @module Tamina
+*/
+@:enum abstract Scheme(String) from String to String  {
```

Comment:
The difference is:

* having `from String` because value can come from external source as a string
* having `from String` because enum is not complete

Former is a valid reason, latter is an invalid reason.